docs/content/manual/_incoming/2577-create-multiple-vm-using-template-EFI-mode.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Create multiple VM instances using VM template with EFI mode selected
+---
+
+* Related issues: [#2577](https://github.com/harvester/harvester/issues/2577) [BUG] Boot in EFI mode not selected when creating multiple VM instances using VM template with EFI mode selected.
+
+## Category: 
+* Virtual Machine
+
+## Verification Steps
+1. Create a VM template, check the `Booting in EFI mode`
+1. Wait for all VM running
```

Comment:
There's no step here where you use the VM template to create multiple VMs

Reply:
Appreciated for the reminder, I add one more steps to create multiple VM instance and use the VM template created with `Booting in EFI mode`